kurtosis-devnet/pectra.yaml

```diff
@@ -68,8 +68,8 @@ optimism_package:
 ethereum_package:
   participants_matrix:
     el:
-      - el_type: geth
-        el_image: ethpandaops/geth:prague-devnet-5-f85cde7
+      - el_type: reth
+        el_image: ethpandaops/reth:main-9001cc2
```

Comment:
Using reth here since they apparently have support for EIP7702 (whereas geth does not yet).

Reply:
The version used for devnet 5 is shown here https://github.com/ethpandaops/pectra-devnets/blob/6f4e7ebd19d35a10392c45facc053602384e0690/ansible/inventories/devnet-5/group_vars/all/images.yaml#L16